src/admin/content-block/helpers/generators/projects-spotlight.ts

```diff
@@ -0,0 +1,73 @@
+import { ImageInfo } from '@viaa/avo2-components';
+
+import i18n from '../../../../shared/translations/i18n';
+
+import { FileUploadProps } from '../../../../shared/components/FileUpload/FileUpload';
+import {
+	ContentBlockBackgroundColor,
+	ContentBlockConfig,
+	ContentBlockEditor,
+	ContentBlockType,
+	DefaultContentBlockState,
+} from '../../content-block.types';
+
+import { times } from 'lodash-es';
+import {
+	CONTENT_BLOCK_FIELD_DEFAULTS,
+	FILE_FIELD,
+	FORM_STATE_DEFAULTS,
+	TEXT_FIELD,
+} from './defaults';
+
+export const INITIAL_PROJECTS_SPOTLIGHT_BLOCK_COMPONENT_STATES = (): ImageInfo[] =>
+	times(
+		3,
+		() =>
+			({
+				image: undefined as any,
+				title: '',
+				buttonAction: {
+					type: 'CONTENT_PAGE',
+				},
+			} as any)
```

Comment:
Is deze `any` an geen `ImageInfo`?

Reply:
ja en nee
doordat we image verplicht maken is die json geen geldige ImageInfo aangezien image nog undefined is, maar eens de gebruiker een image upload zal het wel een geldig ImageInfo object worden.